transcom/mymove
# INT B-21986 v2

[Previous INT PR](https://github.com/transcom/mymove/pull/14316)

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21986)

## Summary

The saga continues.

With the change in the previous PR that fixed the estimated incentive to use the provided estimated weight, that changed the _previously_ implemented change to the SSW form to show the same value, which we do not want. The SSW show should show, under the max obligations, the price that reflects the MAX authorized weight and not the estimated weight, which can be wildly different.

Since we couldn't seem to share these two functionalities, I decided to store the value in the database and use it in the SSW. This will ensure we have different values between the estimated and max allowable.

### How to test

1. Access MM as a customer
2. Create a PPM move
3. Once you see the estimated incentive value 
4. Go check the db and you should see a different value in the `max_incentive` column
5. Finish out your PPM, including closeout
6. Download the payment packet
7. Ensure that the price/value in the db shows under the `Maximum Obligations` section

## Screenshots
After filling out PPM info, you can see the estimated incentive
<img width="638" alt="Screenshot 2024-12-02 at 10 50 12 PM" src="https://github.com/user-attachments/assets/6865fe04-a55a-42bc-be7c-8213f90e8d5d">
`estimated_incentive` in the db
<img width="258" alt="Screenshot 2024-12-02 at 10 50 37 PM" src="https://github.com/user-attachments/assets/46b4237e-b974-42c1-bd6b-b16c99055dc1">

On the SSW
<img width="642" alt="Screenshot 2024-12-02 at 10 52 57 PM" src="https://github.com/user-attachments/assets/13656913-dda2-4248-a98e-e11fa38bdda0">
`max_incentive` in the db
<img width="152" alt="Screenshot 2024-12-02 at 10 53 12 PM" src="https://github.com/user-attachments/assets/33c1198e-6e66-4a9e-b3ad-e203abc37bc1">
